components/esp_hw_support/sleep_modes.c

```diff
@@ -406,11 +403,11 @@ void esp_deep_sleep_deregister_hook(esp_deep_sleep_cb_t old_dslp_cb)
 // [refactor-todo] provide target logic for body of uart functions below
 static void IRAM_ATTR flush_uarts(void)
 {
-    for (int i = 0; i < SOC_UART_NUM; ++i) {
+    for (int i = 0; i < SOC_UART_HP_NUM; ++i) {
```

Comment:
Does it mean that LP uarts are never suspended?

Reply:
I believe so. This is following current implementation for IDF